xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-docker/src/test/it/org/xwiki/image/lightbox/test/ui/LightboxIT.java

```diff
@@ -46,54 +46,37 @@
 
 /**
  * Functional tests for the image lightbox.
- * 
+ *
  * @version $Id$
  * @since 14.1RC1
  */
 @UITest(properties = {
     // Add the FileUploadPlugin which is needed by the test to upload attachment files
-    "xwikiCfgPlugins=com.xpn.xwiki.plugin.fileupload.FileUploadPlugin"})
+    "xwikiCfgPlugins=com.xpn.xwiki.plugin.fileupload.FileUploadPlugin" })
 class LightboxIT
 {
+    public static final String USER_NAME = "JohnDoe";
+
     private static final DocumentReference LIGHTBOX_CONFIGURATION_REFERENCE =
         new DocumentReference("xwiki", Arrays.asList("XWiki", "Lightbox"), "LightboxConfiguration");
 
     private static final String LIGHTBOX_CONFIGURATION_CLASSNAME = "XWiki.Lightbox.LightboxConfigurationClass";
 
     private static final List<String> IMAGES = Arrays.asList("image1.png", "image2.png", "missingImage.png");
 
-    public static final String USER_NAME = "JohnDoe";
-
     @BeforeAll
     void beforeAll(TestUtils testUtils)
     {
         testUtils.createUserAndLogin(USER_NAME, "pa$$word");
+        testUtils.updateObject(LIGHTBOX_CONFIGURATION_REFERENCE, LIGHTBOX_CONFIGURATION_CLASSNAME, 0,
+            "isLightboxEnabled", "1");
     }
 
     @Test
     @Order(1)
-    void disabledLightbox(TestUtils testUtils, TestReference testReference, TestConfiguration testConfiguration)
```

Comment:
Why remove this test? Even if the lightbox is enabled by default we still need to check that an administrator can disable it.